src/components/add-review-form.jsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from 'react';
+
+export const AddReviewForm = () => {
+  const [, setReview] = useState({
+    rating: '',
+    commentText: '',
+  });
+
+  const handleRatingFilm = (evt) => {
+    const { value } = evt.target;
+    setReview((prevReview) => ({ ...prevReview, rating: value }));
+  };
+
+  const handleCommentText = (evt) => {
+    const { value } = evt.target;
+    setReview((prevReview) => ({ ...prevReview, commentText: value }));
+  };
+
+  return (
+    <div className="add-review">
+      <form action="#" className="add-review__form">
+        <div className="rating">
+          <div className="rating__stars">
+            <input onChange={handleRatingFilm} className="rating__input" id="star-1" type="radio" name="rating" value="1"/>
```

Comment:
Let's make it with using array.map

```suggestion
            {
              RATINGS.map((...)) => (
                <input ... />
              )
            }
```

Reply:
готово